valhalla/gui/common_gui.py

```diff
@@ -73,21 +73,21 @@ def get_costing_options(costing_group, profile):
     :rtype: dict
     """
 
-    costing_widgets = costing_group.findChildren((QComboBox, QSpinBox, QDoubleSpinBox))
+    costing_widgets = costing_group.findChildren((QComboBox, QSpinBox, QDoubleSpinBox, QCheckBox))
     costing_widgets = [o for o in costing_widgets if profile in o.objectName()]
 
     costing_options = dict()
     for widget in costing_widgets:
         param_name = widget.objectName().replace(profile + '_', '')
         if isinstance(widget, (QDoubleSpinBox, QSpinBox)):
             if widget.objectName() == 'truck_length' or widget.objectName() == 'truck_width':
-                costing_options[param_name] = round(widget.value() / 3.28084, 2)
+                costing_options[param_name] = widget.value()
```

Comment:
IMO there is no need to keep IF for truck length and width at all so lets just keep only IF for spinboxes

` if isinstance(widget, (QDoubleSpinBox, QSpinBox)):
costing_options[param_name] = widget.value()`

Reply:
Sorry, I left the wrong version of the file during the test.

It should be fixed now.

Thank you very much for the feedback.